tools/nvqpp/nvq++.in

```diff
@@ -404,6 +457,10 @@ while [ $# -ne 0 ]; do
 		MAPPING_FILE="base64_"$(echo -n $2 | base64 --wrap=0)
 		shift
 		;;
+	--codegen-assembly-spec)
```

Comment:
Should we also add this flag to the help string? (Or maybe it is intended as an hidden option)

Reply:
I was leaning towards having it hidden for the moment. I think, for normal users, letting the `--target <foo>` set it is the wiser course.